### ISSUE TITLE
Cannot run git.exe

### ISSUE THREAD

**Issue by Peter-Svahn** [gitextensions/gitextensions#10366]
### Environment

- GitExtensions version: 4.0
- GIT version: 2.8.1
- OS version: Windows 10
- .NET version: 

### Issue description

Git is installed in C:\Program Files\Git\bin, and this is the path shown in settings, git/path. But for every command when git is launched, I get an error message: "C:\Program" is not recognized as an internal or external command. Looks like the path is not quoted correctly when used. 

I have found a workaround by adding git.exe to my path environment variable and changing settings to only git.exe. Then it works.
However, when I open settings again this is changed back to the original above.

Strange that not everyone gets this problem?...

### Steps to reproduce

Run any command, e.g., fetch.

### Did this work in previous version of GitExtensions?

Yes, in 3.5 it worked.

**Comment by mstv** [comment]
These settings may be a little confusing. Have you specified the git _executable_ - not just the folder?
The following works well for me:

![grafik](https://user-images.githubusercontent.com/36601201/200954800-d4079c86-d107-4e10-b4ef-5940326743d9.png)

**Comment by Peter-Svahn** [comment]
Hi, I have used exactly the same settings. This is the same error you would get when launching git from command line with that path without quoting the path. But I don’t see how I can change that.On 9 Nov 2022, at 23:26, Michael Seibt ***@***.***> wrote:﻿
These settings may be a little confusing. Have you specified the git executable - not just the folder?
The following works well for me:

**Comment by Peter-Svahn** [comment]
This how it looks when running any command:

![image](https://user-images.githubusercontent.com/16649725/201187396-aa600949-bd74-405d-8d6c-7c0003db984b.png)

**Comment by Peter-Svahn** [comment]
I tried installing 3.4.3, and that works! Looks like you have a bug here... The dialog window shows that the path to git is quoted in this version.
![image](https://user-images.githubusercontent.com/16649725/201190042-97d23336-3d0e-4ec1-8042-9244b95c199c.png)

**Comment by mstv** [comment]
Update your ancient git, please! Version 2.8 vs. 2.38 is way outdated.
I agree that the error message does not give any hint. I am afraid there is yet another issue.

**Comment by Peter-Svahn** [comment]
Sorry (again), I am using git version 2.38.1.

**Comment by mstv** [comment]
This pathname is always given to the API function without quotes because the arguments are specified using a dedicated argument.
The quotes are just added as a convenience for copying the command - only if the pathname contains a space...

How does it look in the Git Command Log (hotkey `F12`)?

![grafik](https://user-images.githubusercontent.com/36601201/201415886-b9571b06-9235-4df3-9bf5-f0ccec6b69f0.png)

---

You could toggle the usage of ConEmu. But it should not make a difference.

![grafik](https://user-images.githubusercontent.com/36601201/201412776-f12f7d31-ff47-491c-b698-ddf5ece52307.png)

**Comment by Peter-Svahn** [comment]
Disabling the console emulator fixes it! Thanks. Still don't know what the problem is. I did an upgrade from 3.4.3 (I think), and this option was checked by default. I guess more people should get into this problem in that case, or what do you think?

**Comment by RussKie** [comment]
Worth checking that these files aren't blocked by Windows:
![image](https://user-images.githubusercontent.com/4403806/201669370-d00260fb-02cf-48af-8921-716c8dbe7d82.png)

**Comment by Peter-Svahn** [comment]
What do you mean blocked by? How do I check? I could start ConEmu.exe from normal command window.

**Comment by RussKie** [comment]
E.g., 
![image](https://user-images.githubusercontent.com/4403806/202648307-fec6fa8f-992a-45e2-b841-bcc25bda0890.png)

**Comment by Kleidersack** [comment]
Still happens for me on:

 Git Extensions 4.0.2.16100
- Build 25100ec1f7c8da8f798613da74826038af81a50e
- Git 2.39.1.windows.1
- Microsoft Windows NT 10.0.19045.0
- .NET 6.0.13
- DPI 120dpi (125% scaling)
- Microsoft.WindowsDesktop.App Versions

```
    Microsoft.WindowsDesktop.App 3.1.25 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 6.0.13 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 7.0.0 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

**Comment by RussKie** [comment]
Judging by the number of downloads and installations of v4 vs number of reports, this issue is caused by some configuration on your workstation. And you're in best position to debug it and find the root cause.

**Comment by RussKie** [comment]
It doesn't look like this issue is directly attributable to the app itself, rather to something outside of our control.
